src/content/changes/highlights/2021-08-06.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2021-08-06T10:00:00
+title: Highlights for the week ending August 6, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+The unique, immutable cluster identifier formerly called the cluster ID is now called the cluster name, and what was called name is now called description. The goal here is consistency between our user interfaces and the Management API. Likewise for node pools.
+
+## Apps
+
+[Linkerd2 v0.6.0](https://docs.giantswarm.io/changes/managed-apps/linkerd2-app/v0.6.0/), our managed service mesh offering, is now available as an app in the Giant Swarm Catalog. To try it, please read the [documentation](https://github.com/giantswarm/linkerd2-app).
```

Comment:
This PR https://github.com/giantswarm/docs/pull/1073 needs to be merged before link will work

Reply:
Approved. Let's then also wait until https://github.com/giantswarm/docs/pull/1071 picks it up and then approve that one.

Reply:
That's done.

Reply:
Hmmm. Linkerd already disappeared from Playground https://docs.giantswarm.io/changes/playground-apps/, but it's still not in Managed Apps https://docs.giantswarm.io/changes/managed-apps/

Reply:
Ahhh. Just merged this one https://github.com/giantswarm/docs/pull/1075#pullrequestreview-724267825